src/Microsoft.OData.ModelBuilder/Vocabularies/Capabilities/V1/ScopeTypeConfiguration.cs

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+// This is an auto generated file. Please run the template to modify it.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.OData.Edm;
+using Microsoft.OData.Edm.Vocabularies;
+
+namespace Microsoft.OData.ModelBuilder.Capabilities.V1
+{
+	/// <summary>
+	/// Org.OData.Capabilities.V1.ScopeType
+	/// </summary>
+	public partial class ScopeTypeConfiguration : IRecord
```

Comment:
>partial [](start = 8, length = 7)

the reason to mark the class as partial

Reply:
To enable extensions that are not autogenerate-able.